subprojects/smoke-test/build.gradle.kts

```diff
@@ -84,7 +84,7 @@ tasks {
         group = "Verification"
         testClassesDirs = smokeTestSourceSet.output.classesDirs
         classpath = smokeTestSourceSet.runtimeClasspath
-        maxParallelForks = 1 // those tests are pretty expensive, we shouldn't execute them concurrently
+        maxParallelForks = 2 // those tests are pretty expensive, we shouldn't execute them concurrently
```

Comment:
I suppose the comment needs adjusting as well?

Reply:
Done.